src/amo/reducers/abuse.js

```diff
@@ -156,7 +156,8 @@ type Action =
 
 export default function abuseReducer(
   state: AbuseState = initialState,
-  action: Action,
+  // $FlowIgnore
+  action: Action = {},
```

Comment:
Copying from previous PR:
> Not sure how best to handle that. This is because eslint (rightly) complains that default values should be last, but redux wants them in that order... I don't think we care about the action in that case...

Reply:
I reverted that and went in the opposite direction in a7eac127d25bfd9fa88e6f11ea58a4ee4918022a, disabling the eslint rule for the `state` parameter in reducers